decidim-conferences/config/locales/en.yml

```diff
@@ -331,7 +331,7 @@ en:
           invite:
             decline: Decline invitation '%{conference_title}'
             invited_you_to_join_a_conference: "%{invited_by} has invited you to join a conference at %{application}. You can decline or accept it through the links below."
-            join: Join conference '%{conference_title}'
+            registration: Registration '%{conference_title}'
```

Comment:
"Registration Decidim JAM" sounds very weird to me, I'd remove the conference title or change it to "Registration for X"

Reply:
@decidim/product we need your opinion here

Reply:
Where does this go? If it's the Registration button leave only "Registration"

Reply:
Nop. It is for the mail that user recieves when she Register to a Conference

Reply:
Oh ok, in that case let's use Registration for '%{conference_title}'

Reply:
Changed